actions/project.js

```diff
@@ -21,6 +21,13 @@ const {
     getAioLogger, getUrlInfo, handleExtension, getFloodgateUrl, getDocPathFromUrl
 } = require('./utils');
 
+const PROJECT_STATUS = {
+    NOT_STARTED: 'NOT STARTED',
+    COMPLETED: 'COMPLETED',
+    COMPLETED_WITH_ERROR: 'COMPLETED WITH ERROR',
+    IN_PROGRESS: 'IN PROGRESS'
```

Comment:
Added a new constant IN PROGRESS in addition to the ones we got on milo project.